weave-js/src/components/Panel2/PanelFilterEditor.tsx

```diff
@@ -92,10 +92,15 @@ type VisualClause =
   | VisualClauseNumber;
 
 interface VisualClauseWorkingState {
-  key: string | undefined;
-  simpleKeyType: 'number' | 'string' | 'boolean' | 'other';
-  op: string | undefined;
-  value: string | string[] | number | boolean | undefined;
+  key?: string | undefined;
+  simpleKeyType?: 'number' | 'string' | 'boolean' | 'other';
+  op?: string | undefined;
+  value?:
+    | string
+    | number
+    | boolean
+    | Array<string | number | boolean>
```

Comment:
This PR should be doable without expanding the type here. Only the string filter allows an IN query right now, so we should always have Array<string> here. If we store something else here, we've done something wrong! (so should be handled be earlier asserts instead of expanding the type).

I'm cool to merge this since this as a user facing improvement, but I think it'd be good to do a little followup to narrow the type back down.